lib/Doctrine/ORM/Persisters/Collection/ManyToManyPersister.php

```diff
@@ -498,8 +498,13 @@ protected function getInsertRowSQL(PersistentCollection $collection)
             $types[]   = PersisterHelper::getTypeOfColumn($joinColumn['referencedColumnName'], $targetClass, $this->em);
         }
 
+        $ignore = '';
+        if ($mapping['ignoreDuplicates']) {
+            $ignore = 'IGNORE'; // @todo
```

Comment:
I'll appreciate guidance here: I assume I need to open a PR for DBAL that will allow platforms specyfying the keyword that needs to be used (or return `null` if it's not supported)?

Reply:
Maybe yeah… I see that there is a `AbstractPlatform::getForUpdateSQL()` method that looks designed that way.

Reply:
So looking at https://github.com/doctrine/dbal/blob/012569dfcc96478c061d3b529fecb5c8ac0405e5/src/Platforms/SqlitePlatform.php#L626-L629 the method would return empty string if the platform does not support `INSERT IGNORE` and generated query would be a casual `INSERT`, right?

Reply:
Yup, sounds right 👍